### ISSUE TITLE
A11y_SQLAzureDataStudio_Query_Create&Run_ScreenReader: Screen reader does not read the content present inside results grid.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9472]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#DesktopApp;#MacOs;#Benchmark;#A11y_SQLAzureDataStudioBenchmark;#A11ySev1;#MAS1.3.1;#Voiceover;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
MAS reference: MAS1.3.1
Mac OS:10.15.3

**Repro Steps:**
1.Launch Azure Data Studio.
2.Navigate to File > New Query
3.Connect to a Server in the Query Editor and choose the credential, and optionally the database to connect to the server.
4.Type a SQL query statement.
5.Click Run.
6.Navigate to Results section.
7.Check if all content present under results grid is read by screen reader.

**Actual:**
The content present below the results grid is not read by screen reader.

**Expected:**
All the content present on the page should be read by screen reader.

**User Impact:**
Screen reader users will not receive all the information that is present on the screen.

**Attachment link reference:**
[9472_A11y_SQLAzureDataStudio_Query_Create&Run_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/4296638/9472_A11y_SQLAzureDataStudio_Query_Create.Run_ScreenReader.zip)

**Comment by aniketr99** [comment]
Verified the bug in below environment:
Version: 1.41.1 (Universal)
Commit: https://github.com/microsoft/azuredatastudio/commit/33dce95b152227c4f10f6251ab30690fe9895c0c
Date: 2023-01-28T00:49:19.454Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 22.3.0
issue is fixed hence adding #Closed; tag.

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-SQLADS-Win32-Feb2020;
